### PR TITLE
Prepare v0.43.0 release

### PR DESCRIPTION
**Pull request by philipaconrad**
Version 0.43.0 is almost ready to ship!

Review/comments appreciated. :+1:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,82 @@
...
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 0.43.0
+
+This release contains a number of fixes and enhancements.
+
+### Tooling, SDK, and Runtime
+
+- Add line count to test coverage report (#4843) (authored by @FarisR99)
```
Comment (ashutosh-narkar) [inline CHANGELOG.md:12]: Do you need to add link to #4843 and the rest? Are they auto-rendered ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,82 @@
...
+- build: run 'make generate' in release container
```
Comment (ashutosh-narkar) [inline CHANGELOG.md:13]: We can probably move this to the misc section with a some description about why this was done.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,7 +3,82 @@
...
+- Bundles fixes, changes:
+  - plugins: accept any HTTP 2xx for status and logs (#4897) (authored by @lvisterin)
+  - runtime: Treat other Unix-es like macos (#4931) (authored by @iamleot)
```
Comment (ashutosh-narkar) [inline CHANGELOG.md:16]: A little description will help.

File: CHANGELOG.md (hunk 1 above)
Comment (philipaconrad) [reply CHANGELOG.md:12]: I don't know. Those are just what the `make release-patch` script coughed up. I'll take a look at what we've done for previous releases, and alter as needed.

File: CHANGELOG.md (hunk 3 above)
Comment (philipaconrad) [reply CHANGELOG.md:16]: Right. I think the patch-generator script just chucks over the exact commit message. Some edits for cleanup+consistency could go a long way here.

File: CHANGELOG.md (hunk 2 above)
Comment (srenatus) [reply CHANGELOG.md:13]: We might even drop this, it's not a user-visible change.

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,7 +3,82 @@
...
+- website: fix community page issues (#4904) (authored by @mstade)
+
+- Ecosystem Additions:
+  - Rond (authored by @ugho16)
+  - walt.id
+
+### Miscellaneous
+
+- Dependency bumps, notably:
+  - aquasecurity/trivy-action from 0.5.1 to 0.6.0
```
Comment (srenatus) [inline CHANGELOG.md:70]: I'd drop the CI-related ones.

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,7 +3,82 @@
...
+  - ast/builtins: fix 'type_name' type and docs
+  - topdown/copypropagation: keep refs into livevars (#4936) ([#4848](https://github.com/open-policy-agent/opa/issues/4848)) reported by @johanneskra
+  - topdown/parse_units: Use big.Rat for units parsing. (#4857) ([#4856](https://github.com/open-policy-agent/opa/issues/4856)) reported by @tmos22
+  - topdown: is_valid functions no longer error. (#4844) ([#4760](https://github.com/open-policy-agent/opa/issues/4760))
+  - topdown: support glob.match without delimiters (#4933) ([#4923](https://github.com/open-policy-agent/opa/issues/4923)) authored by @vinhph0906 reported by @vinhph0906
+- ast: move schema tests from compile_test.go to schema_test.go
+
+### Documentation
+
+ - docs/bundles: add bundle CLI command documentation (#4881) ([#3831](https://github.com/open-policy-agent/opa/issues/3831)) authored by @Joffref
```
Comment (srenatus) [inline CHANGELOG.md:44]: I usually remove the `(#...)` bit -- it's a thing github adds to commits when using squash and merge.

File: CHANGELOG.md (hunk 3 above)
Comment (philipaconrad) [reply CHANGELOG.md]: I have reworded/rewritten the descriptions for all of the PRs to better reflect what changed.

File: CHANGELOG.md (hunk 1 above)
Comment (philipaconrad) [reply CHANGELOG.md]: Edits are up, and I have stripped out those auto-generated `(#...)` entries.

File: CHANGELOG.md (hunk 4 above)
Comment (philipaconrad) [reply CHANGELOG.md]: I'll strip them out shortly...

File: CHANGELOG.md (hunk 5 above)
Comment (philipaconrad) [reply CHANGELOG.md]: Done. :+1:

File: CHANGELOG.md (hunk 4 above)
Comment (philipaconrad) [reply CHANGELOG.md]: Done. :+1:

File: CHANGELOG.md (hunk 2 above)
Comment (philipaconrad) [reply CHANGELOG.md]: It is done. :+1:

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,7 +3,96 @@
...
+  - Server: Old policy path is now checked for bundle ownership before update ([#4846](https://github.com/open-policy-agent/opa/issues/4846))
+  - Storage+Bundle: Old bundle data is now cleaned before new bundle activation ([#4940](https://github.com/open-policy-agent/opa/issues/4940))
+  - Bundle: Paths are now normalized before bundle root check occurs
+
+#### Storage Fixes
+
+The Storage system received mostly bugfixes, with a notable performance improvement for large bundles in this release:
+
+  - storage/inmem: Speed up bundle activation by avoiding unnecessary read operations ([#4898](https://github.com/open-policy-agent/opa/issues/4898))
+  - storage/disk: Fixed a mutex locking around an error condition
```
Comment (ashutosh-narkar) [inline CHANGELOG.md:41]: Nit: We can omit this.

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,7 +3,95 @@
...
```
Comment (ashutosh-narkar) [inline CHANGELOG.md:41]: Nit: this can be removed I think.

File: CHANGELOG.md [hunk 8]
```diff
@@ -3,7 +3,95 @@
...
+
+#### Bundles Fixes
+
+The Bundles system received several bugfixes and performance improvements in this release:
+
+  - Bundle: `opa bundle` command now supports `.yml` files ([#4859](https://github.com/open-policy-agent/opa/issues/4859)) authored by @Joffref reported by @rdrgmnzsakt
+  - Plugins/Bundle: Use unique temporary files for persisting activated bundles to disk ([#4782](https://github.com/open-policy-agent/opa/issues/4782)) authored by @FredrikAppelros reported by @FredrikAppelros
+  - Server: Old policy path is now checked for bundle ownership before update ([#4846](https://github.com/open-policy-agent/opa/issues/4846))
+  - Storage+Bundle: Old bundle data is now cleaned before new bundle activation ([#4940](https://github.com/open-policy-agent/opa/issues/4940))
+  - Bundle: Paths are now normalized before bundle root check occurs
```
Comment (ashutosh-narkar) [inline CHANGELOG.md:34]: what about: `Paths are now normalized before bundle root check occurs to ensure checks are os-independent`

File: CHANGELOG.md [hunk 9]
```diff
@@ -3,7 +3,95 @@
...
+
+  - storage/inmem: Speed up bundle activation by avoiding unnecessary read operations ([#4898](https://github.com/open-policy-agent/opa/issues/4898))
+  - storage/disk: Fixed a mutex locking around an error condition
+  - storage/inmem: Paths are now created during truncate operations if they did not exist before (#4853)
+  - storage/disk: Symlinks work with relative paths now (#4870) ([#4869](https://github.com/open-policy-agent/opa/issues/4869))
+
+### Rego and Topdown
+
+The Rego compiler and runtime environment received a number of bugfixes, and a few new features this release, as well as a notable performance improvement for large Objects
+(covered below).
```
Comment (ashutosh-narkar) [inline CHANGELOG.md:48]: `covered above` 😄

File: CHANGELOG.md [hunk 10]
```diff
@@ -3,7 +3,95 @@
...
+### Documentation
+
+ - Docs: Add GraphQL API authorization tutorial
+ - Docs/bundles: Add bundle CLI command documentation (#4881) ([#3831](https://github.com/open-policy-agent/opa/issues/3831)) authored by @Joffref
+ - Docs/policy-reference: Remove extra quote in Grammar to fix formatting (#4916) ([#4915](https://github.com/open-policy-agent/opa/issues/4915)) authored by @friedrichsenm reported by @friedrichsenm
+ - Docs/policy-testing: Add missing future.keywords imports (#4852) ([#4849](https://github.com/open-policy-agent/opa/issues/4849)) reported by @robert-elles
+ - Docs: Add note about counter_server_query_cache_hit metric (#4946) ([#4389](https://github.com/open-policy-agent/opa/issues/4389))
+ - Docs: Kube tutorial includes updated cert install procedure (#4907) ([#4902](https://github.com/open-policy-agent/opa/issues/4902)) reported by @Imp
+ - Docs: GraphQL builtins section now includes a note about framework-specific `@directive` definitions in GraphQL schemas
+ - Docs: Replace OPA docker image tag with shortcode
```
Comment (ashutosh-narkar) [inline CHANGELOG.md:69]: Nit: this can be removed I think.

File: CHANGELOG.md [hunk 11]
```diff
@@ -3,7 +3,95 @@
...
+ - Docs: Add warning about name collisions in older policies from importing 'future.keywords'
+
+### Website + Ecosystem
+
+- Website: Show navbar on smaller devices ([#3353](https://github.com/open-policy-agent/opa/issues/3353)) authored by @Parsifal-M reported by @OBrienCommaJosh
+- Website/frontpage: Update examples
```
Comment (ashutosh-narkar) [inline CHANGELOG.md:75]: Can we add some detail here or remove this?

File: CHANGELOG.md [hunk 12]
```diff
@@ -3,7 +3,95 @@
...
+- Website/live-blocks: Only pass 'import future.keywords' when needed and supported
+- Website/live-blocks: Update codemirror-rego to 1.3.0
+- Website: Fix community page issues (#4904) (authored by @mstade)
```
Comment (ashutosh-narkar) [inline CHANGELOG.md:78]: Can we add some detail here ?

**Review by ashutosh-narkar — APPROVED** [review]
This looks great! Just a few comments.

File: CHANGELOG.md (hunk 7 above)
Comment (philipaconrad) [reply CHANGELOG.md:41]: I have removed the note about the mutex lock error condition item.

File: CHANGELOG.md (hunk 11 above)
Comment (philipaconrad) [reply CHANGELOG.md:75]: It is done. :+1:

File: CHANGELOG.md (hunk 10 above)
Comment (philipaconrad) [reply CHANGELOG.md:69]: It is done. :+1:

File: CHANGELOG.md (hunk 9 above)
Comment (philipaconrad) [reply CHANGELOG.md:48]: Good catch! Thanks! :+1:

File: CHANGELOG.md (hunk 8 above)
Comment (philipaconrad) [reply CHANGELOG.md:34]: I like the extra context that adds. I'm putting that in presently. :+1:

File: CHANGELOG.md (hunk 12 above)
Comment (philipaconrad) [reply CHANGELOG.md:78]: Details added. Good catch! :+1:

**Comment by philipaconrad** [comment]
Changes have been squashed locally. Should be ready for merge!

**Review by ashutosh-narkar — APPROVED** [review]

**Comment by philipaconrad** [comment]
Will merge when green. Thanks @ashutosh-narkar! :+1:

**Pull request merged**